Commands.md

```diff
@@ -226,11 +226,11 @@ This is how the `tebular` output style and `oneline` output style look like
 
 ```bash
 $ git summary --output-style tabular
-# Repo     | Age       | Last active | Active on | Commits | Uncommitted
```

Comment:
We need to update the non-output-style version too.

Reply:
Sorry. This slipped through. It is not updated. I double checked all locations and the `branch` is not contained everywhere.